installers-conda/resources/post-install.sh

```diff
@@ -65,6 +76,16 @@ To uninstall Spyder, you need to run from the following from the command line:
 
 $ uninstall-spyder
 
+#####################
+# !!! IMPORTANT !!! #
+#####################
+
+The spyder and uninstall-spyder command-line shortcuts will only be available
+in new shell sessions. To make them available in this session you must source
+your .bashrc file
```

Comment:
```suggestion
The spyder and uninstall-spyder commands will only be available in new shell
sessions. To make them available in this session you must source your .bashrc
file with:
```

* I think `shortcuts` sounds a bit strange in a Linux context, but perhaps you used it because you wrote `command-line` before that term. So, I suggest to remove `command-line shortcuts` and just leave `commands` instead.
* Add `with` at the end of this paragraph to make it a bit easier to read.

Reply:
Unless you think `command-line aliases` is better. But I'm fine with your suggestion as well.

Reply:
I thought about that too, but I think the simpler this message the better (I guess several users don't know what an `alias` is).